pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/OptionAstNode.java

```diff
@@ -54,4 +55,33 @@ public void updateBrokerRequest(BrokerRequest brokerRequest) {
 
     brokerRequest.getQueryOptions().put(left, right);
   }
+
+  @Override
+  public void updatePinotQuery(PinotQuery pinotQuery) {
+    if (getChildren().size() != 2) {
```

Comment:
Can `getChildren()` return null?

Reply:
Most of the new code is based on existing code, it's just populating PinotQuery instead of BrokerRequest. Since we don't have a null check in the existing code, we don't need one here.